core/complex/inspect_spec.rb

```diff
@@ -13,4 +13,11 @@
     Complex(-1, -4).inspect.should == "(-1-4i)"
     Complex(-7, -6.7).inspect.should == "(-7-6.7i)"
   end
+
+  it "uses parentheses for Rationals" do
```

Comment:
This is confusing because the usage of parentheses is regardless of rationals or not.

Reply:
Ah you mean the inner parentheses.
Yeah that's just Rational# to_s vs inspect.
Indeed it'd be much better to test that with a mock than depending on the details of some other method.

Reply:
Well, it took some tries to actually create a Complex with the mocks, but I guess the spec is now correct.
The addition of `*` based on the last character of the stringification of the imaginary part has been based on the C implementation of MRI.